src/lib/browse/layers/lines/CyclePaths.svelte

```diff
@@ -132,4 +100,41 @@
   {/each}
 {/if}
 
-<InteractiveLayer layer={name} {tooltip} {show} clickable on:click={onClick} />
+<VectorTileSource
+  url={`pmtiles://${publicResourceBaseUrl()}/v1/${name}.pmtiles`}
+>
+  <LineLayer
+    {...layerId(name)}
+    sourceLayer={name}
+    paint={{
+      "line-color": constructMatchExpression(
+        ["get", "kind"],
+        {
+          track: colors.cycle_paths.track,
+          lane: colors.cycle_paths.lane,
+          shared_use_segregated: colors.cycle_paths.shared_use_segregated,
+          shared_use_unsegregated: colors.cycle_paths.shared_use_unsegregated,
+        },
+        "red"
+      ),
+      "line-width": denseLineWidth,
+      "line-opacity": hoverStateFilter(1.0, 0.5),
+    }}
+    layout={{
+      visibility: show ? "visible" : "none",
+    }}
+    hoverCursor="pointer"
+    on:click={onClick}
+  >
+    <Popup let:props>
+      {@const [kind, direction, width] = tooltip(props)}
```

Comment:
This is a weird hybrid style for tooltips. Do the bulk of the logic in a function, but format in Svelte